metadata/schemas/sensor.json

```diff
@@ -51,32 +51,22 @@
                 "type": {
                     "type": "string",
                     "enum": [
-                        "temperature", "displacement", "radioactivity", "humidity"
+                        "temperature",
+                        "displacement",
+                        "radioactivity",
+                        "humidity"
```

Comment:
I thought you wanted to change that to relative_humidity? In case that is done, please mark the comment as resolved.